applications/luci-app-tailscale/htdocs/luci-static/resources/view/tailscale/setting.js

```diff
@@ -0,0 +1,265 @@
+
+		o = s.taboption('advance', form.Flag, 'acceptRoutes', _('Accept Routes'), _('Accept subnet routes that other nodes advertise.'));
+		o.default = o.disabled;
+		o.rmempty = false;
+
+		o = s.taboption('advance', form.Value, 'hostname', _('Device Name'), _("Leave blank to use the device's hostname."));
+		o.default = '';
+		o.rmempty = true;
+
+		o = s.taboption('advance', form.Flag, 'acceptDNS', _('Accept DNS'), _('Accept DNS configuration from the Tailscale admin console.'));
+		o.default = o.enabled;
+		o.rmempty = false;
+
+		o = s.taboption('advance', form.Flag, 'advertiseExitNode', _('Exit Node'), _('Offer to be an exit node for outbound internet traffic from the Tailscale network.'));
+		o.default = o.disabled;
+		o.rmempty = false;
+
+		o = s.taboption('advance', form.ListValue, 'exitNode', _('Online Exit Nodes'), _('Select an online machine name to use as an exit node.'));
+		if (onlineExitNodes.length > 0) {
+			o.optional = true;
+			onlineExitNodes.forEach(function(node) {
+				o.value(node, node);
+			});
+		} else {
+			o.value('', _('No Available Exit Nodes'));
+			o.readonly = true;
+		}
+		o.default = '';
+		o.depends('advertiseExitNode', '0');
+		o.rmempty = true;
+
+		o = s.taboption('advance', form.DynamicList, 'advertiseRoutes', _('Expose Subnets'), _('Expose physical network routes into Tailscale, e.g. <code>10.0.0.0/24</code>.'));
+		if (interfaceSubnets.length > 0) {
+			interfaceSubnets.forEach(function(subnet) {
+				o.value(subnet, subnet);
+			});
+		}
+		o.default = '';
+		o.rmempty = true;
+
+		o = s.taboption('advance', form.Flag, 's2s', _('Site To Site'), _('Use site-to-site layer 3 networking to connect subnets on the Tailscale network.'));
+		o.default = o.disabled;
+		o.depends('acceptRoutes', '1');
+		o.rmempty = false;
+
+		o = s.taboption('advance', form.DynamicList, 'subnetRoutes', _('Subnet Routes'), _('Select subnet routes advertised by other nodes in Tailscale network.'));
+		if (subnetRoutes.length > 0) {
+			subnetRoutes.forEach(function(route) {
+				o.value(route, route);
+			});
+		} else {
+			o.value('', _('No Available Subnet Routes'));
+			o.readonly = true;
+		}
+		o.default = '';
+		o.depends('s2s', '1');
+		o.rmempty = true;
+
+		o = s.taboption('advance', form.MultiValue, 'access', _('Access Control'));
+		o.value('tsAClan', _('Tailscale access LAN'));
+		o.value('tsACwan', _('Tailscale access WAN'));
+		o.value('lanACts', _('LAN access Tailscale'));
+		o.value('wanACts', _('WAN access Tailscale'));
+		o.default = "tsAClan tsACwan lanACts";
+		o.rmempty = true;
+
+		s.tab('extra', _('Extra Settings'));
+
+		o = s.taboption('extra', form.DynamicList, 'flags', _('Additional Flags'), String.format(_('List of extra flags. Format: --flags=value, e.g. <code>--exit-node=10.0.0.1</code>. <br> %s for enabling settings upon the initiation of Tailscale.'), '<a href="https://tailscale.com/kb/1241/tailscale-up" target="_blank">' + _('Available flags') + '</a>'));
```

Comment:
Maybe break it up into several lines and use template literals to make this more readable.